go.mod

```diff
@@ -277,3 +278,5 @@ replace github.com/openshift/oc-mirror/v2/pkg/cli => ./v2/pkg/cli
 replace github.com/google/go-containerregistry => github.com/google/go-containerregistry v0.10.0
```

Comment:
Instead of manually vendoring and putting in a risk of old oc is being used, you can use replace;
```
replace(
github.com/google/go-containerregistry => github.com/google/go-containerregistry v0.10.0
github.com/openshift/oc v0.0.0-alpha.0.0.20230609203739-4e136762ec0e => github.com/ardaguclu/oc introduce-refreshable-creds-factory
)
```

`go mod tidy` and `go mod vendor` will automatically update vendor directory and fetch the required changes.

Reply:
Never mind, you have already added a log `Code based on PR #1540 https://[github.com/openshift/oc/pull/1540](http://github.com/openshift/oc/pull/1540)` indicates the code is used or not.